src/Oscoin/Storage/ContentHashStore.hs

```diff
@@ -0,0 +1,73 @@
+-- | 'ContentHashStore' is an interface for storing values adressed by
+-- their hashes. Conceptually 'ContentHashStore' is a mutable 'Map'
+-- with hashes as keys.
+module Oscoin.Storage.ContentHashStore
```

Comment:
More name bikeshedding: can we simply go for `HashStore`? The "Content" there doesn't really add any information.

Reply:
Sounds like Görlitzer Park 😆

Reply:
:joy: